spec/pantry_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper'
+
+RSpec.describe Kitchen::Pantry do
+  let(:instance) { described_class.new }
+
+  it 'creates a hash' do
```

Comment:
the constructor creates a pantry object, not a hash, so need to change this spec name